external-crates/move/crates/move-compiler/src/unit_test/plan_builder.rs

```diff
@@ -491,12 +489,9 @@ fn parse_failure_attribute(
     }
 }
 
-fn parse_gas_budget_attribute(context: &mut Context, gas_budget_attr: Option<&E::Attribute>) -> Option<GasBudgetParams> {
-    let mut assigned_gas_budget = GasBudgetParams {
-        compute_budget:1,
-        heap_size: 1,
-        max_call_depth: 1,
-    };
+fn parse_gas_budget_attribute(context: &mut Context, gas_budget_attr: Option<&E::Attribute>) -> Option<SolanaGasBudgetParams> {
+
```

Comment:
Empty line at the beginning of the function body, likely would be removed by an automatic formatter.